### DISCUSSION TITLE
Additions to the documentation.

### DISCUSSION THREAD

**Discussion by ChronoGit** [joshreve/dactyl-keyboard#3] [Q&A]
First of all, I want to thank you for this project, finally making the Dactyl Customization more accessible, especially with the STEP files.

One question I still have, however, is what connectors are used for this version of the Manuform. The Link to the RJ9s seems dead, and the type of USB connector that fits the hole is not mentioned. Would it be possible to add pictures pertaining to this to the documentation?

Thank you,
Chrono

**Comment by joshreve** [comment]
Sorry for the late response, I didn't have bandwidth for a bit and am just getting back to it.  The USB hole frankly just expects a lot of hot glue around whatever you bring out.  Not very fancy.  I did get some standard RJ9 from amazon that worked.  I'll update the link. . . . . forget that, the one I used is no longer available.  I like the RJ9 better than the TRRS as it locks in, but it looks like they are getting more scarce these days.  These look like the ones I got, but it looks like they are hard to find now.  [RJ9 at Walmart?](https://www.walmart.com/ip/2Pcs-Female-Telephone-Connector-Adapter-616E-4P4C-RJ9-w-15cm-4-Colorful-Cables/114461595?wmlspartner=wlpa&selectedSellerId=571)
